tests/testthat/test-ss.R

```diff
@@ -41,3 +41,16 @@ test_that("ss_n and ss_fixed issue-533", {
   expect_warning(mrgsim_e(mod,e, ss_n=3), 
                  "\\[steady_zero\\] ID 246 failed to reach steady state")
 })
+
+test_that("ss tolerances", {
+  mod <- house()
+  expect_is(mod@ss_rtol,"numeric")
+  expect_is(mod@ss_atol,"numeric")
+  dose <- ev(amt = 100, ii = 24, addl = 0, ss = 1)
+  out1 <- mrgsim_e(mod,dose,ss_rtol = 1e-3)
+  out2 <- mrgsim_e(mod,dose,ss_rtol = 1e-6)
+  out3 <- mrgsim_e(mod,dose,ss_rtol = 1e-6)
```

Comment:
Can we test default value here by setting `out3 <- mrgsim_e(mod, dose)`, can we?

Reply:
Sure; I'll add that.